src/tools/one_version/allowlist.cc

```diff
@@ -67,4 +67,14 @@ absl::flat_hash_set<std::string> MapAllowlist::AllLabels(
   return allowlist_[package_name];
 }
 
+std::unique_ptr<Allowlist> MapAllowlist::Parse(std::istream& in) {
+  absl::flat_hash_map<std::string, absl::flat_hash_set<std::string>> map;
+  std::string line;
+  while (std::getline(in, line)) {
+    std::vector<std::string> parts = absl::StrSplit(line, absl::MaxSplits(' ', 1));
+    map[parts[0]].insert(parts[1]);
+  }
```

Comment:
This segfaults if it's given an allowlist that doen't have at least two records on each line, could you add some rudimentary error handling here?

Also I have a slight preference for moving the parsing logic to main, to keep MapAllowlist agnostic of the allowlist format we're using here.